_MyGet.cmd

```diff
@@ -0,0 +1,111 @@
+@echo off
```

Comment:
This is a duplicated `build.cmd` file without running the tests. Could you please remove this, and add a `-skiptests` option to `build.cmd` instead?